<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CategoryManager</class>
 <widget class="QWidget" name="CategoryManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CategoryManager</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
CategoryManager { background-color: #e8f5e9; }
QGroupBox { background-color: white; border: 1px solid #c8e6c9; border-radius: 8px; margin-top: 10px; padding: 10px; font-weight: bold; }
QLineEdit, QListWidget { background-color: #f9fff9; border: 1px solid #a5d6a7; border-radius: 4px; padding: 4px; color: black; }
QPushButton { background-color: #43a047; color: white; font-weight: bold; border-radius: 6px; padding: 6px 10px; }
QPushButton:hover { background-color: #2e7d32; }
QLabel { color: black; }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="headerLayout">
     <item>
      <widget class="QPushButton" name="homeBtn">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>üè†</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>üìÇ Manage Categories</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignHCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="rightSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="categoryGroup">
     <property name="title">
      <string>New / Edit Category</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Category Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="categoryNameInput"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="addCategoryBtn">
       <property name="text">
        <string>‚ûï Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateCategoryBtn">
       <property name="text">
        <string>‚úèÔ∏è Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteCategoryBtn">
       <property name="text">
        <string>üóë Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="categoryListWidget">
     <property name="styleSheet">
      <string notr="true">QListWidget::item:selected {
    background-color: #81c784;   /* brighter green for visibility */
    color: white;
    border-radius: 4px;
}

QListWidget::item:hover {
    background-color: #a5d6a7;
}</string>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
